vividus-plugin-web-app-playwright/src/main/java/org/vividus/ui/web/playwright/steps/RequestValidationSteps.java

```diff
@@ -0,0 +1,151 @@
+     *                       <li><b>REQUEST_DATA</b> - the request body and the response status code
+     *                       <li><b>RESPONSE_DATA</b> - the response body
+     *                       </ul>
+     * @param scopes         The set (comma separated list of scopes e.g.: STORY, NEXT_BATCHES) of variable's scope<br>
+     *                       <i>Available scopes:</i>
+     *                       <ul>
+     *                       <li><b>STEP</b> - the variable will be available only within the step,
+     *                       <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     *                       <li><b>STORY</b> - the variable will be available within the whole story,
+     *                       <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     *                       </ul>
+     * @param variableName The variable name to save the URL.
+     */
+    @When("I capture HTTP $httpMethods request with URL pattern `$urlPattern` and save $httpMessagePart to $scopes "
+            + "variable `$variableName`")
+    public void captureRequestAndSaveHttpMessagePart(Set<String> httpMethods, Pattern urlPattern,
+            HttpMessagePart httpMessagePart, Set<VariableScope> scopes, String variableName)
+    {
+        List<Request> matchedRecords = findRecords(httpMethods, urlPattern);
+        String description = String.format("Number of HTTP %s requests matching URL pattern %s",
+                methodsToString(httpMethods), urlPattern);
+        softAssert.assertThat(description, matchedRecords, hasSize(equalTo(1)));
+
+        if (matchedRecords.size() == 1)
+        {
+            Request request = matchedRecords.get(0);
+            variableContext.putVariable(scopes, variableName, httpMessagePart.get(request));
+        }
+    }
+
+    /**
+     * Waits for appearance of HTTP request matched <b>httpMethod</b> and <b>urlPattern</b> in requests archive.
+     * @param httpMethods The comma-separated HTTP methods to filter by, e.g. 'GET,POST,PUT'
+     * @param urlPattern  The regular expression to match HTTP request URL
+     */
+    @When("I wait until HTTP $httpMethods request with URL pattern `$urlPattern` exists in requests archive")
+    public void waitRequestInRequestsArchive(Set<String> httpMethods, Pattern urlPattern)
+    {
+        BrowserContext browserContext = browserContextProvider.get();
+
+        BooleanSupplier waitCondition = () -> !findRecords(httpMethods, urlPattern).isEmpty();
+        String assertionMessage = String.format("waiting for HTTP %s request with URL pattern %s",
+                methodsToString(httpMethods), urlPattern);
+
+        waitActions.runWithTimeoutAssertion(assertionMessage, () -> browserContext.waitForCondition(waitCondition));
+    }
+
+    /**
+     * Clears the requests archive.
+     */
+    @When("I clear requests archive")
```

Comment:
```suggestion
    @When("I clear network recordings)
```
and also please change the syntax for Selenium proxy steps (the old syntax should be deprecated)